packages/core/useFileDialog/index.ts

```diff
@@ -0,0 +1,52 @@
+import { readonly, ref } from 'vue-demi'
+
+export interface UseFileDialogOptions {
+  multiple?: boolean
+  accept?: string
+}
+
+const DEFAULT_OPTIONS: UseFileDialogOptions = {
+  multiple: true,
+  accept: '*',
+}
+
+/**
+ * Open file dialog with ease.
+ *
+ * @see https://vueuse.org/useFileDialog
+ * @param options
+ */
+export function useFileDialog(options?: Partial<UseFileDialogOptions>) {
+  const files = ref<FileList | null>(null)
+
+  const open = (localOptions?: Partial<UseFileDialogOptions>) => {
+    const _options = {
+      ...DEFAULT_OPTIONS,
+      ...options,
+      ...localOptions,
+    }
+    const input = document.createElement('input')
```

Comment:
Is it possible to move to the `useFileDialog` scope? I'm worried that using the `open` function multiple times will create too many `input`.

Reply:
Moved outside `useFileDialog` now. Thanks!